### PR TITLE
Fixed display error and added display for no schedule

### LINKED ISSUE #192: Readjust Display when Schedule is Hidden
image

### PR DESCRIPTION
**Pull request by qiud1**
Closes #192 

Now properly hides the schedule when there is no current one and displays if there is. If there is no schedule, displays a template error message.

image

image

### CODE REVIEW

**Comment by qiud1** [comment]
The two images are separate screenshots

File: src/components/Schedule.vue [hunk 1]
```diff
@@ -1,9 +1,12 @@
 <template>
-  <b-card v-if="!isFsMode && isCurrent" class="mt-3" :class="[{'bubble-dark': isDarkMode},{'bubble-light': !isDarkMode}]">
-    <h3 :class="{'text-white': isDarkMode}">
+  <b-card v-if="!isFsMode" class="mt-3" :class="[{'bubble-dark': isDarkMode},{'bubble-light': !isDarkMode}]">
+    <h3 v-if="!isCurrent" :class="{'text-white': isDarkMode}">
+      No Current Schedule
+    </h3>
+    <h3 v-if="isCurrent" :class="{'text-white': isDarkMode}">
```
Comment (Isaac-the-Man) [inline src/components/Schedule.vue:6]: Use ternary operator on the text instead of doing this.

File: src/components/Schedule.vue [hunk 2]
```diff
@@ -65,21 +68,9 @@ export default {
...
-
-        //assign current schedule and enable display
-        if (Date.parse(this.schedules[idx]["end"]) <= Date.parse(Date()) &&
-            Date.parse(this.schedules[idx]["start"]) >= Date.parse(Date()))
-        {
-          this.currentWeek = this.schedules[idx]["content"]
-          return true
-        } 
-      })
+      if (!this.currentWeek) { return false}
```
Comment (Isaac-the-Man) [inline src/components/Schedule.vue:71]: Replace with `return this.currentWeek === undefined`

File: src/components/Schedule.vue [hunk 3]
```diff
@@ -96,6 +87,8 @@ export default {
 
       const current = new Date();
 
+      this.currentWeek = null
```
Comment (Isaac-the-Man) [inline src/components/Schedule.vue:90]: unnecessary code. Default is value `undefined` already.

**Review by Isaac-the-Man — CHANGES_REQUESTED** [review]

**Comment by qiud1** [comment]
Made requested changes.

File: src/components/Schedule.vue (hunk 3 above)
Comment (qiud1) [reply src/components/Schedule.vue]: The default value was not undefined, so I have set it to undefined.

File: src/components/Schedule.vue [hunk 4]
```diff
@@ -20,36 +20,7 @@ export default {
...
-				"start": "01:00",
-				"end": "23:00"
-			},
-			"sunday": {
-				"start": "01:00",
-				"end": "23:00"
-			}
-		},
+      currentWeek: undefined,
       currentSemester: "Spring 2022"
```
Comment (Isaac-the-Man) [inline src/components/Schedule.vue:24]: Set `currentSemester` to `undefined`.

**Review by Isaac-the-Man — CHANGES_REQUESTED** [review]

**Review by Isaac-the-Man — APPROVED** [review]

**Pull request merged**